### ISSUE TITLE
npm install for submodules skipped

### ISSUE THREAD

**Issue by semenovDL** [wojtekmach/acme_bank#3]
For correct work of this example you also need to install packages for phoenix applications:

```
$ mix ecto.setup
$ cd apps/bank_web
$ npm install
$ cd ../backoffice/
$ npm install
$ cd ../..
$ mix phoenix.server
```

**Comment by wojtekmach** [comment]
Good catch, I'll update setup instructions

**Comment by bchase** [comment]
Ha, I just submitted a pull request (#5) for this... then spotted this issue!

**Comment by wojtekmach** [comment]
Closed by #5
